source/diagrams/diagrams.txt

```diff
@@ -34,6 +34,13 @@ rules.
 The relational view is unaffected by mapping rules, and only changes
 when you update your project's data model.
 
+.. tip::
+
+  You can also use `Hackolade <https://hackolade.com/>`__ to 
```

Comment:
Nit -- should it be "Hackolade" or "Hackolade Polyglot Data Modeling"? Their site is unclear since the latter dominates the page, but Hackolade is the tool name given in the window title.

Reply:
Several references seem to refer to the tool as "Hackolade" such as their [Github](https://github.com/hackolade):


```
Hackolade
Visual and agile data modeling for NoSQL databases and REST APIs: https://github.com/mongodb, https://github.com/neo4j, @apachecassandra, https://github.com/Swagger, https://github.com/couchbase, @avro, and many more to come
```
Given this I am inclined to leave this as is.